editor/components/editor-global-keyboard-shortcuts/index.js

```diff
@@ -64,6 +64,7 @@ class EditorGlobalKeyboardShortcuts extends Component {
 		const { hasMultiSelection, clearSelectedBlock } = this.props;
 		if ( hasMultiSelection ) {
 			clearSelectedBlock();
+			window.getSelection().removeAllRanges();
```

Comment:
Should this be a side-effect (e.g. `store/effects.js`) of the `CLEAR_SELECTED_BLOCK` action?